### ISSUE TITLE
Accessibility tab needs an indicator

### ISSUE THREAD

**Issue by arjunkalburgi** [storybookjs/storybook#12220]
**Describe the bug**
The accessibility tab needs an indicator when there is a problem.
<img width="311" alt="image" src="https://user-images.githubusercontent.com/8989414/91064118-d8cc2c00-e5fc-11ea-864b-bd68797f6026.png">

**To Reproduce**

1. Make a colour contrast error in your component (black text on black background)
2. Click on the Controls tab in storybook

There is no way people will know there's an accessibility error unless they check the tab. This is bad UX. please include the number of error just like the Controls tab has.

**Comment by jsomsanith** [comment]
This can be considered as duplicate of #16235
